tool/release/notes/validate_release_notes.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Copyright (C) 2022 Vaticle
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+if git -C $BUILD_WORKSPACE_DIRECTORY diff --exit-code HEAD^ HEAD -- LATEST_RELEASE_NOTES.md; then
```

Comment:
This command checks whether the diff of `LATEST_RELEASE_NOTES.md` since the last commit is empty.
`--exit-code` makes `git diff` exit with `1` if there were any differences, and `0` if there were none.